skipper.env

```diff
@@ -1,3 +1,5 @@
+BUILD_ID
+build_id
```

Comment:
Why in lower case and why it's duplicated?

Reply:
The CI jobs use lowercase parameters while test-infra uses uppercase. This makes both of them work.

Reply:
I think that you should export then in upper case. 
@osherdp WDYT?

Reply:
probably, if it's possible. there's also a chance it depends on other components
@michaellevy101 is it possible?

Reply:
yes, It's. but i think supporting they both is simpler